composer.json

```diff
@@ -16,8 +16,8 @@
         "jms/translation-bundle": "^1.3"
     },
     "require-dev": {
-        "phpunit/phpunit": "^5.7|^6.0",
-        "matthiasnoback/symfony-dependency-injection-test": "~1.0|~2.0",
+        "phpunit/phpunit": "^5.7|^6.0|^7.5",
```

Comment:
Does it really make sense to still have this `|`? It's require-dev package, so in a deterministic way only a specific PHPUnit version will be installed and I expect it to be `^7.5`, because it supports all the PHP versions in the test matrix. Keep in mind that `|` causes more memory usage for Composer (1.x)

Reply:
Fixed in https://github.com/ezsystems/repository-forms/pull/339/commits/02720f8bfd981ebb96d00c227ec1dbd1e97b84e3